app/gui/language/ast/impl/src/crumbs.rs

```diff
@@ -1465,16 +1465,16 @@ pub trait TraversableAst: Sized {
     }
 
     /// Calculate the span of the descendent AST node described by given crumbs..
-    fn span_of_descendent_at(&self, crumbs: &[Crumb]) -> FallibleResult<Span> {
-        let mut position = Index::new(0);
+    fn span_of_descendent_at(&self, crumbs: &[Crumb]) -> FallibleResult<text::Range<Bytes>> {
```

Comment:
The name of the method might need updating, as it no longer returns a `Span` but a `Range`.